meshes/ur30/visual/upperarm.dae

```diff
@@ -0,0 +1,418 @@
+<?xml version="1.0" encoding="utf-8"?>
+<COLLADA xmlns="http://www.collada.org/2005/11/COLLADASchema" version="1.4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <asset>
+    <contributor>
```

Comment:
- [ ] Please change the contributor xml section to match the UR20 dae files with:

```
<contributor>
      <author>Universal Robots A/S</author>
      <copyright>2023 Universal Robots A/S</copyright>
      <comments>Use hereof is subject to Universal Robots A/S’ Terms and Conditions for Use of Graphical Documentation</comments>
</contributor>
```